navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/NavigationService.java

```diff
@@ -79,8 +77,7 @@ public void setupNotification(Activity activity) {
     // Sets up the top bar notification
     notifyBuilder = new NotificationCompat.Builder(this)
       .setContentTitle("Mapbox Navigation")
-      .setContentText("Distance: " + routeProgress.getCurrentLegProgress().getCurrentStepProgress()
-        .getDistanceRemaining())
+      .setContentText("navigating")
```

Comment:
Can we provide a more descriptive text by default? Is there an API for the dev to update this text? (can be ticketed separately)